src/components/Property/RaiseWorkOrder/AddMultipleSORs.js

```diff
@@ -0,0 +1,39 @@
+import { TextArea, PrimarySubmitButton } from '../../Form'
+import { useForm } from 'react-hook-form'
+
+const AddMultipleSORs = ({ setCurrentPage }) => {
+  const { register, handleSubmit, errors } = useForm()
+  const onSubmit = () => {}
+  const FORM_PAGE = 1
+  return (
+    <>
+      <a
+        className="govuk-back-link lbh-back-link govuk-!-display-none-print"
+        role="button"
+        onClick={() => setCurrentPage(FORM_PAGE)}
+      >
+        Back
+      </a>
+      <h1 className="lbh-heading-h1 govuk-!-margin-bottom-2">
+        Add multiple SOR codes
+      </h1>
+      <form
+        role="form"
+        id="adding-multiple-sors-form"
+        onSubmit={handleSubmit(onSubmit)}
+      >
+        <TextArea
+          name="note"
+          label="Enter SOR codes as a list:"
+          register={register({
+            required: 'Please add notes',
```

Comment:
This message should reference SOR codes. There's no specified message in the ticket but I suggest we use 'Please enter SOR codes'. I didn't mention it earlier but please could we also include a check in the Cypress tests for the presence validation message after submitting an empty form.